.github/workflows/libxrpl.yml

```
@@ -27,7 +27,7 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
           running-workflow-name: wait-for-check-regexp
-          check-regexp: '(dependencies|test).*linux.*' # Ignore windows and mac tests but make sure linux passes
+          check-regexp: '(dependencies|test).*linux.*' # Ignore Windows and MacOS tests, but make sure Linux tests pass
```

Official capitalization used by [Apple](https://www.apple.com/macos)
```suggestion
          check-regexp: '(dependencies|test).*linux.*' # Ignore Windows and macOS tests, but make sure Linux tests pass
```